Makefile

```diff
@@ -117,8 +118,9 @@ build-cli-%: ##Build the Tanzu Core CLI for a platform
 	else \
 		GOOS=$(OS) GOARCH=$(ARCH) $(GO) build --ldflags "$(LD_FLAGS)"  -o "$(ARTIFACTS_DIR)/$(OS)/$(ARCH)/cli/core/$(BUILD_VERSION)/tanzu-cli-$(OS)_$(ARCH)" ./cmd/tanzu/main.go;\
 	fi
-
-
+	mkdir -p bin
+	cp $(ARTIFACTS_DIR)/$(OS)/$(ARCH)/cli/core/$(BUILD_VERSION)/tanzu-cli-$(OS)_$(ARCH) ./bin/tanzu
```

Comment:
I think this should only be done if the os/arch are the ones matching the local computer.  Or dependingelse "make cross-build" will create as bin/tanzu that may not work on the local machine.

Reply:
moved under `build` target